tensorflow/core/kernels/sequence_ops.cc

```diff
@@ -93,8 +93,16 @@ class RangeOp : public OpKernel {
     }
     int64_t size;
```

Comment:
@cantonios Should we cast size as a `uint64_t`?

Reply:
Do you mean "declare"?  Probably.

Reply:
+1 on declaring it as that

Reply:
ok, i can make the change when i'm at my computer this afternoon